src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,666 @@
++++
+title = "User Guide"
+date = 2020-05-04T19:01:14+05:30
+weight = 15
++++
+
+## Overview
+
+This guide is intended for users who have a Submariner environment set up and want to verify the installation and learn more about how to
+use Submariner and the main capabilities it provides. This guide assumes that there are two Kubernetes clusters,
+**cluster-2** and **cluster-3**, forming a clusterset, and that the Broker is deployed into a separate cluster **cluster-1**.
+
```

Comment:
Might add: Make sure you have `subctl` set up (because Submariner could be set up using Helm, though not recommended)